cmd/templates/main.go

```diff
@@ -35,41 +37,61 @@ func main() {
 	flag.Parse()
 
 	// Setup logger.
-	logger := log.New(os.Stderr, "[templatesApi] ", log.Ltime)
+	logger := log.New(os.Stderr, "[templatesApi][server] ", log.Ltime)
+
+	// Envs.
+	envs, err := env.FromOs()
+	if err != nil {
+		logger.Println("cannot load envs: " + err.Error())
+		os.Exit(1)
+	}
 
 	// Start DataDog tracer.
-	tracer.Start(tracer.WithServiceName("templates-api"), tracer.WithLogger(ddLogger{logger}))
-	defer tracer.Stop()
+	if envs.Get("DATADOG_ENABLED") != "false" {
+		tracer.Start(tracer.WithServiceName("templates-api"), tracer.WithLogger(ddLogger{logger}))
+		defer tracer.Stop()
+	}
+
+	// Start server
+	start(*httpHostF, *httpPortF, *debugF, logger, envs)
+}
 
-	// Initialize the services.
-	templatesSvc := templatesApi.NewTemplates(logger)
+func start(host, port string, debug bool, logger *log.Logger, envs *env.Map) {
+	// Create dependencies.
+	ctx, cancel := context.WithCancel(context.Background())
+	d := dependencies.NewContainer(ctx, debug, logger, envs)
+
+	// Log options.
+	d.Logger().Infof("starting HTTP server, host=%s, port=%s, debug=%t", host, port, debug)
```

Comment:
Aby som ukazal ze funguje logger, mozeme to dat potom prec.
![image](https://user-images.githubusercontent.com/19371734/158322544-25ea7a3a-4e42-4ba5-af20-5ce834e1e167.png)